examples/append_conc_to_h5file.py

```diff
@@ -0,0 +1,11 @@
+from pyspecdata import *
+import h5py
+filename = '220126_Ras_M67R1a_capProbe.h5'
+actual_conc = 72e-6 #M
+h5 = search_filename(f"{filename}" , exp_type = 'ODNP_NMR_comp/ODNP')
+h5 = ['/'.join(path.split('\\')) for path in h5]
```

Comment:
what is this doing?

Reply:
h5.File requires / NOT \ so I need to convert

Reply:
fixed using normpath